README.md

```diff
@@ -586,6 +586,111 @@ crud.len('customers')
 ...
 ```
 
+### Statistics
+
+`crud` routers provide statistics on called operations.
+```lua
+-- Returns table with statistics information.
+crud.stats()
+```
+
+Statistics collect enabled by default on all routers.
+It can be disabled and re-enabled later.
+```lua
+-- Disables statistics collect and resets all collectors.
+crud.disable_stats()
+
+-- Enable statistics collect and recreates all collectors.
+crud.enable_stats()
+```
+
+If [`metrics`](https://github.com/tarantool/metrics) found,
+you can use metrics collectors to store statistics
+instead of local collectors.
+It is required to use version `0.9.0` or greater,
+otherwise local collectors will be used.
+```
+-- Use metrics collectors.
+crud.enable_stats({ driver = 'metrics' })
+```
+By default, local collectors (`{ driver = 'local' }`)
+are used. Metrics collectors are much sophisticated and
+would show execution time quantile with aging for calls.
+Be wary that computing quantiles may affect overall
+performance under high load. Using local
```

Comment:
"performance under high load" looks strange.

I can't imagine "performance without high load" and why it won't affected.

Reply:
Rewritten the text